qchem/Data/Enums.py

```diff
@@ -3,6 +3,7 @@
 # Enum for Orca Basis Sets
 class OrcaBasisSet(Enum):
     DEF2_SVP = "DEF2-SVP"
+    MINI = "MINI"
 
 # Enum for Orca Calculation Types
 class OrcaCalculationType(Enum):
```

Comment:
We should be a little cautious about adding too many overlapping things here--not much gained having both OPTIMIZEPARALLEL and BASICPARALLEL (with calc=OPT). Might run into headaches later when we get to having a lot of these